src/components/PatientsTable/PatientsTable.tsx

```diff
@@ -0,0 +1,59 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { type Patient } from '@medplum/fhirtypes'
+import {
+  DataTable,
+  type DataTableProps,
+} from '@stanfordspezi/spezi-web-design-system/components/DataTable'
+import { createColumnHelper } from '@tanstack/table-core'
+import { PatientMenu } from './PatientMenu'
+
+interface PatientsDataTableProps
+  extends Omit<DataTableProps<Patient>, 'columns'> {
+  onRowClick?: (data: Patient) => void
+  onDelete: (data: Patient) => void
+  editRoute: string
```

Comment:
`editRoute` cannot be a plain `string`. Edit routes usually contain ID, so it should be:

```
(patientId: string) => string
```